erpnext/accounts/doctype/bank_account/bank_account.py

```diff
@@ -135,6 +135,9 @@ def get_party_bank_account(party_type, party):
 
 
 def get_default_company_bank_account(company, party_type, party):
+	if party_type not in ["Customer", "Supplier", "Company"]:
```

Comment:
Why company?

Reply:
It should not return for other party_type. Instead, it should skip fetching the default bank account from the party master if party_type is not customer/supplier.